src/Console/TranslateExtract.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Rdosgroup\GptTranslate\Console;
+
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class TranslateExtract extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $signature = 'translate:extract {--origin=en} {--lang=sv} {--model=gpt-3.5-turbo}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Get all Translatable stringa and translate to a given language into a .json file';
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $this->info('Creating translation file for ' . $this->option('lang') . ' at ' . Carbon::now()->toDateTimeString());
```

Comment:
Should use laravel prompts :)

https://laravel.com/docs/10.x/prompts
https://github.com/laravel/prompts

Its installed in L10 by default